docs/index.rst

```diff
@@ -12,26 +12,55 @@
 
 ----------------------------------
 
-.. include:: quickstart.rst
-    :start-line: 1
+.. toctree::
+   :maxdepth: 1
+   :caption: Concepts
 
+   concepts/tutor 
+   concepts/openedx
+   
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
    :caption: User guide
 
-   gettingstarted
+   install
+   deployment
    run
    configuration
    plugins/index
-   reference/index
-   tutorials/index
+   analytics
+   sysadmin/index
+   cli
    troubleshooting
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Developer guide
+
+   developing/nightly
    tutor
-   faq
+   developing/plugins/index
```

Comment:
Move developing/plugins/index before tutor/ (one line above) -- because plugin development is more important than core development.

Reply:
addressed here https://github.com/overhangio/tutor/compare/7eb67a7705d82a0038855bc1003f95499c2b3470..8c2e147a5ed97ee19580e2049c7fd252894fcd93